requirements/test.txt

```diff
@@ -1,7 +1,7 @@
 # requirements for running the tests via pytest
 freezegun==0.3.11
 mock==2.0.0
-pytest>=3.6
+pytest<5.0.0>=3.6
```

Comment:
new version of pytest(5.0.0) do not run with pytest-django

Reply:
Add this comment to the file